pkg/api/event.proto

```diff
@@ -239,13 +259,15 @@ message EventMessage {
     }
 }
 
+// The cause for a job to error.
 enum Cause {
     Error = 0;
     Evicted = 1;
     OOM = 2;
     DeadlineExceeded = 3;
 }
 
+// A Docker containers status.
```

Comment:
Not sure if this is true.  This is the container status and Docker is one container runtime for kubernetes.  

I would just leave this as A container status.